### DISCUSSION TITLE
无法链接新终端

### DISCUSSION THREAD

**Discussion by Fu1249519541** [Eugeny/tabby#6577] [Q&A]
无法新建和重新链接终端，必须关闭Tabby,重新打开才可以
![{$WSTR)`4N1S1)QT_FUCE2C](https://user-images.githubusercontent.com/86100946/172782901-b1f9a01c-627d-4327-8994-00e067f50d22.png)
。

**Comment by lawweiliang** [comment]
@Fu1249519541 
English only please.
https://github.com/Eugeny/tabby/discussions/5533

![image](https://user-images.githubusercontent.com/34936929/228172727-cc58dcc4-bdf4-4085-844a-a25cf784c145.png)
